dietpi/dietpi-software

```diff
@@ -2971,6 +2979,21 @@ _EOF_
 
 		fi
 
+		software_id=190 # Beets
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			G_EXEC mkdir /etc/beets
+			G_EXEC chmod 777 /etc/beets
+			
+			G_EXEC eval "echo -e '#!/bin/dash\nexport BEETSDIR=/etc/beets' > /etc/bashrc.d/dietpi-beets.sh"
```

Comment:
No need for the shebang. Using dash is even confusing as `/etc/bashrc.d` of course is only loaded for interactive bash shells. As it is sourced, not executed, no shebang is required. Not sure if we use a shebang in other such cases, but then it should be removed.